index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Tweeter</title>
+    <link rel="shortcut icon" type="image/jpg" href=./images/icon.png/>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <header><img alt="tweeter_icon" class="tweeter_icon" src="images/icon.png">Tweeter<img alt="tweeter_icon" class="tweeter_icon" src="images/icon.png"></header>
+    <br>
+    <div id="main_div">
+        <input id="txt_twit" type="text" placeholder="What's in your mind?">
+        <button type="button" id="btn_twit">Twit</button>
+    </div>
+    <div id="container">
+        <!--
```

Comment:
remove `//comments` and `console.log()`s when you finish your code

Reply:
shall I remove all comments or just the important comments? to make my code more readable